mrp_production_service/models/mrp_production.py

```diff
@@ -24,15 +24,15 @@ def _create_service_procurement(self, line):
         return self.env['procurement.rule'].create(data)
 
     @api.model
-    def _action_launch_procurement_rule(self, line, production):
-        values = self._prepare_service_procurement_values(production)
+    def _action_launch_procurement_rule(self, line):
+        values = self._prepare_service_procurement_values()
 
         name = '%s for %s' % (line.product_id.name,
-                              production.name)
+                              self.name)
         self.env['procurement.group'].sudo().run(
-            line.product_id, line.product_qty,
+            line.product_id, self.product_qty,
```

Comment:
if the MO is for 3 and the parent/child ratio for product/service is 1:2 in the BoM, this line will procure for 3 (as you use `self.product_qty`) units of service instead of 6, am I wrong?

Reply:
It seems to be fixed, I also update test